src/interface/src/app/shared/form-message-box/form-message-box.component.html

```diff
@@ -0,0 +1,10 @@
+<div *ngIf="message" class="message-box" [ngClass]="getBoxClass()">
```

Comment:
message is optional, so potentially this will not render anything.
I think the component should always be shown internally and let the parent control its visibility

Reply:
Right, good catch, this is leftover from a previous approach